src/util.jl

```diff
@@ -288,4 +293,92 @@ function shiftin!{T}(a::Vector{T}, b::Vector{T})
 end
 
 
+
+# Delay Utility Functions:
+
+# finddelay(): use peak of cross correlation from origin of time to calculate
+# time lag between two Arrays (signals)
+
+function finddelay{T <: Real}(x::AbstractArray{T, 1}, u::AbstractArray{T, 1})
+
+sₓᵤ = xcorr(x, u)
```

Comment:
The convention in the rest of the package is to indent the content of functions.